actor/src/main/scala/org/apache/pekko/actor/dungeon/Children.scala

```diff
@@ -147,8 +147,8 @@ private[pekko] trait Children { this: ActorCell =>
 
   @nowarn private def _preventPrivateUnusedErasure = {
     _childrenRefsDoNotCallMeDirectly
-    _functionRefsDoNotCallMeDirectly
     _nextNameDoNotCallMeDirectly
+    _functionRefsDoNotCallMeDirectly
```

Comment:
Anz reason behind this reordering?

Reply:
I updated this (to revert the order change)